st2common/st2common/runners/base.py

```diff
@@ -288,7 +288,11 @@ def pre_run(self):
                                                   entry_point=self.entry_point,
                                                   worktree_path=self.git_worktree_path)
 
-            assert(entry_point.startswith(self.git_worktree_path))
+            if not entry_point.startswith(self.git_worktree_path):
+                raise ValueError('The entry point value "%s"' %
+                                 'does not match with git worktree path "%s".' %
```

Comment:
I don't think this will do what you want it to do.

```suggestion
                raise ValueError('The entry point value "%s" '
                                 'does not match with git worktree path "%s".' %
```